client/styles/welcome.scss

```diff
@@ -1,3 +1,150 @@
+@import url("https://fonts.googleapis.com/css2?family=Heebo:wght@400;700&display=swap");
+
 .welcome-page {
+  position: relative;
+  width: 375px;
+  height: 800px;
+  background-color: rgba(78, 63, 40, 1);
+}
+
+.top {
+  background-color: white;
+  height: 500px;
+  display: flex;
+}
+
+#open-pic {
+  background: url(/client/assets/images/yaar.png);
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  width: 239px;
+  height: 239px;
+  // ask what's the exact distance "margin-top"
+  margin-top: 30px;
+  margin-left: 68px;
+}
+
+#website-title {
+  // ask if to use with flex or text-align, cause it's noe shorter
+  position: absolute;
+  width: 308px;
+  height: 75px;
+  margin-top: 22px;
+  left: 36px;
+  top: 275px;
+  font-family: "Heebo";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 59px;
+  color: rgba(0, 0, 0, 1);
+}
+
+#sub-title {
+  position: absolute;
+  margin-top: 0px;
+  width: 307px;
+  height: 38px;
+  top: 355px;
+  left: 37px;
+  font-family: "Heebo";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 26px;
+  line-height: 38.19px;
+  color: #000000;
+}
+
+#btn-login {
+  background-color: #3b5a16;
+  border-radius: 5px;
+  margin-top: 34px;
+  margin-left: 135px;
+  width: 105px;
+  height: 37px;
+  top: 380px;
+  font-family: "Heebo";
+  font-weight: 400;
+  font-size: 14px;
+  font-style: normal;
+  line-height: 20.56px;
+  color: #ffffff;
+}
+
+#btn-login:hover {
+  background-color: rgba(209, 210, 174, 1);
+  color: #000000;
+}
+
+#sign-in {
+  position: absolute;
+  width: 44px;
+  height: 21px;
+  margin-left: 165px;
+  top: 465px;
+  font-family: "Heebo";
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 20.56px;
+  color: #000000;
   text-align: center;
+  text-decoration-line: underline;
+}
+
+#sign-in:hover {
+  font-weight: bold;
+}
+
+#welcome {
```

Comment:
welcome what? button? text? rename this to be clearer

Reply:
ok